doc/user_manual/existing_interfaces.tex

```diff
@@ -2620,3 +2622,327 @@ \subsubsection{DataObjects}
+\end{lstlisting}
+   that will be converted in the following way (CSV):
+    \begin{table}[h]
+    \centering
+    \caption{CSV History Overview}
+    \label{CSVhistoryOverview}
+    \tabcolsep=0.11cm
+    \tiny
+    \begin{tabular}{|c|c|c|c|c|c|c|c|}
+    \hline
+     time    & t0  & t1      & dt      & flux & fluence & power & energy \\  
+     1.0E-06 & 0.0 & 1.0E-06 & 1.0E-06 & 0.0  & 0.0     & 0.0   & 0.0      
+    \end{tabular}
+   \end{table}
+   
+   \item \textit{\textbf{concentration tables}} 
+  \begin{lstlisting}[basicstyle=\tiny]
+=========================================================================================================================
+=   Nuclide concentrations in watts, actinides for case 'decay' (#1/1)                                                  =
+-------------------------------------------------------------------------------------------------------------------------
+  (relative cutoff; integral of concentrations over time >   1.00E-04 % of integral of all concentrations over time)
+.
+                0.0E+00sec  1.0E-06sec  
+  th231       8.6167E-08  8.6167E-08  
+  th234       7.7763E-09  7.7763E-09 
+------------
+  totals       4.6831E+03  4.6831E+03  
+=========================================================================================================================
+.
+.
+=========================================================================================================================
+=   Nuclide concentrations in watts, fission products for case 'decay' (#1/1)                                           =
+-------------------------------------------------------------------------------------------------------------------------
+  (relative cutoff; integral of concentrations over time >   1.00E-04 % of integral of all concentrations over time)
+.
+                0.0E+00sec  1.0E-06sec  
+  ga74        2.4264E-01  2.4264E-01  
+  ga75        1.8106E+00  1.8106E+00  
+------------
+  totals       1.2266E+06  1.2266E+06 
+  \end{lstlisting}
+  that will be converted in the following way (CSV):
+   \begin{table}[h]
+    \centering
+    \caption{CSV Concentration Tables}
+    \label{CSVconcentrationTables}
+    \tabcolsep=0.11cm
+    \tiny
+    \begin{tabular}{|c|c|c|c|c|c|c|c|}
+    \hline
+     time    & ga74\_watts  & ga75\_watts      & subtotals\_fission\_products      & th231\_watts & th234\_watts & subtotals\_actinides & totals\_watts \\ \hline
+     0.0E+00 & 2.4264E-01 & 1.8106E+00 & 1.2266E+06 & 8.6167E-08  & 7.7763E-09     & 4.6831E+03   & 1.2313E+06    \\ 
+     1.0E-06 & 2.4264E-01 & 1.8106E+00 & 1.2266E+06 & 8.6167E-08  & 7.7763E-09     & 4.6831E+03  & 1.2313E+06      
+    \end{tabular}
+   \end{table}
+\end{itemize}
+
+\textbf{Remember also that a SCALE simulation run is considered successful (i.e., the simulation did not crash) if it does not contain, in 
+the last 20 lines, the following message:}
```

Comment:
can we check through all the file for this error instead of considering the last 20 lines? I recall something similar for the RELAP5 interface

Reply:
In case of scale, that message is always printed at the end since it is printed by the main driver.
I use the last 20 lines to avoid to parse the all files (super slow in case of big file)